packages/apollo-server-fastify/src/__tests__/ApolloServer.test.ts

```diff
@@ -0,0 +1,835 @@
+import { FastifyInstance } from 'fastify';
+import fastify from 'fastify';
+
+import http from 'http';
+
+import request from 'request';
+import FormData from 'form-data';
+import fs from 'fs';
+import { createApolloFetch } from 'apollo-fetch';
+
+import { gql, AuthenticationError, Config } from 'apollo-server-core';
+import { ApolloServer, ServerRegistration } from '../ApolloServer';
+
+import {
+  atLeastMajorNodeVersion,
+  testApolloServer,
+  createServerInfo,
+} from 'apollo-server-integration-testsuite';
+
+const typeDefs = gql`
+  type Query {
+    hello: String
+  }
+`;
+
+const resolvers = {
+  Query: {
+    hello: () => 'hi',
+  },
+};
+
+const port = 8888;
```

Comment:
I truly can't blame you for perpetuating the existing patterns which put each server integration test suite on a different port number:

https://github.com/apollographql/apollo-server/blob/ae69675aaae296d0d4cfa0045a7c636486c132c2/packages/apollo-server-hapi/src/__tests__/ApolloServer.test.ts#L16

...but we should probably bump to 9999 instead since Gatsby apps frequently run on this port.  Myself or anyone should tweak this before merge.

Reply:
https://github.com/apollographql/apollo-server/pull/1971/commits/dc0911afc63903a39ff9bf799dd4777a30f2c1aa